### PR TITLE
Fix bugs found in integration test

### PR DESCRIPTION
**Pull request by liuzhe-lz**

### CODE REVIEW

File: pipelines/full-test-linux.yml [hunk 1]
```diff
@@ -15,36 +15,33 @@ jobs:
       echo "##vso[task.setvariable variable=PATH]${PATH}:${HOME}/.local/bin"
       echo "##vso[task.setvariable variable=NNI_RELEASE]999.$(date -u +%Y%m%d%H%M%S)"
 
-      python3 -m pip install -U --upgrade pip setuptools
-      python3 -m pip install -U pytest
+      python3 -m pip install --upgrade pip setuptools
```
Comment (SparkSnail) [inline pipelines/full-test-linux.yml:18]: why remove `-U`?

File: pipelines/integration-test-openpai-linux.yml [hunk 2]
```diff
@@ -26,37 +26,30 @@ jobs:
...
-      nnictl package install .
+      nnictl algo register --meta meta_file.yml
     displayName: Install customized tuner
 
   - script: |
       set -e
       docker login -u nnidev -p $(docker_hub_password)
       echo '## Build docker image ##'
-      docker build --build-arg NNI_RELEASE=${NNI_RELEASE} -t nnidev/nni-it-pai:latest .
+      docker build --build-arg NNI_RELEASE=${NNI_RELEASE} -t nnidev/nni-nightly .
```
Comment (SparkSnail) [inline pipelines/integration-test-openpai-linux.yml:50]: nightly?

File: pipelines/full-test-linux.yml (hunk 1 above)
Comment (liuzhe-lz) [reply pipelines/full-test-linux.yml:18]: Because it's duplicate to `--upgrade`

File: pipelines/integration-test-openpai-linux.yml (hunk 2 above)
Comment (liuzhe-lz) [reply pipelines/integration-test-openpai-linux.yml:50]: Each day scheduled pipelines will build and push several docker images with same content. So it's effectively the nightly version.
I'm planning to write a pipeline to trigger all IT pipelines, which will build and push docker image at first. This is a preparation for that.

**Review by SparkSnail — APPROVED** [review]

**Review by ultmaster — APPROVED** [review]

**Pull request merged**